### PR TITLE
EPF: Fix electronic ISSNs and ISBNs

### PR DESCRIPTION
**Pull request by maccabeelevine**
Although the EDS and EPF APIs both return their identifiers within a structure like 

```
 "RecordInfo": {
   "BibRecord": {
     "BibEntity": {
       "Identifiers": [
         {
           "Type": "issn-print",
           "Value": "00991333"
         },
```

it turns out that the Type metadata is not the same.  Both use `isxn-print` for print.  But for digital items, EPF uses `isxn-online` instead of EDS's `isxn-electronic`.  Go figure.

### CODE REVIEW

**Comment by maccabeelevine** [comment]
Will add a test case...

**Comment by maccabeelevine** [comment]
> Will add a test case...

Done.

**Comment by maccabeelevine** [comment]
FYI @cwolfebsco this is the sort of area where the documentation might be improved.  The EBSCO Connect page for [Publication Finder search](https://connect.ebsco.com/s/article/Publication-Finder-API-Reference-Guide-Search) links to one "Publication Finder JSON Get Search Response, which downloads a file ironically named "EDS_APIJSON_GET_Search_response" (even though it's not EDS).  But ignoring the name, the only ISBN identifiers included isbn-print.  So no way to know the full response schema and the valid values for identifier type.

I'm guessing the new developer preview site may have the full response schema and details like this, which would be great.  It seems to be down at the moment, and in any case I have the preview URL for the EDS API but not EPF.

But even if it's listed there, it might be useful to have a page of things like this that seem like they should be identical between the two APIs, and are almost identical, but actually aren't (-online vs -electronic) for no immediately obvious reason.

**Review by demiankatz — APPROVED** [review]
Thanks, @maccabeelevine, this looks reasonable to me. I'll hold off on merging this until you say you're ready (in case you'd like to hear from @cwolfebsco before finalizing everything), but please feel free to hit the merge button yourself whenever you're comfortable unless you make dramatic changes that you'd like me to re-review.

**Comment by demiankatz** [comment]
Also note that this may be a candidate for backporting or rebasing to the release-10.0 branch if you'd like to get the fix released sooner -- but I'll leave that to your discretion as well. :-)

**Comment by maccabeelevine** [comment]
Sure, I'll give it 24h in case @cwolfebsco wants to weigh in on anything.

Re: backporting, although it is a bug, it's low severity (only affects people actually doing something with the ISSN, which the built-in themes do not), and I may be the only one even using the EPF integration so far.  So I don't think it's worth it.  But I wish we had some telemetry!

**Comment by cwolfebsco** [comment]
@maccabeelevine  Sorry for being consistently late to the party, I will pass on the comment about documentation to Matt for his consideration.

**Comment by maccabeelevine** [comment]
> @maccabeelevine Sorry for being consistently late to the party, I will pass on the comment about documentation to Matt for his consideration.

@cwolfebsco Not late at all!  Very happy to wait a day for business hours in your timezone to get your input.  Thanks again.

**Pull request merged**
